### PR TITLE
added main menu for the game 

### PR DESCRIPTION
**Pull request by kamishettysreeja25**
@niccokunzmann 
I created the main menu for the game which consists of links to 
1-new game-which redirects to  index.html,
2-controls which contains the instructions to play the game which redirects to howtoplay.html, 
3-scores which contains the leader board/scores which  redirects to  leaderboard.html,
4-credits which redirects to the credits.html,
5-Exit which redirects to links.html
I did not work on the styling part as of now and would like to work on enhancement and functionality of different pages.
I am new to this organisation and thought this would address the issue #21 
I would like to know your opinion on the main page and in which way i can contribute to this organisation

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Cool. What I will need to merge it:
- [ ] consistent indentation in heads
- [ ] close every tag you open (also html)
- [X] favicon used by html files
- [X] titles in html files
- [ ] use Links instead if JavaScript wherever you can to open pages. In this case: remove `main_menu.js`

What I would like and add as an issue but you do not need to do:
- [ ] Use SVG files instead of bitmap images because they work on every screen size.

**Comment by kamishettysreeja25** [comment]
@niccokunzmann 
Thanks a lot , Learning from mistakes is the best way to improve.Your response really gave me so much energy.I would really love to work on all the requirements one by one and solve all the errors 
1-I will use text/svg images for the main menu and increase the look of the main menu
2-I will use links.js for the redirection and remove main_menu.js and use nice design principles 
If anything else need to be done please suggest me. 
do we need anything more in the main menu other than new game,controls,credits,scores,exit?

**Comment by niccokunzmann** [comment]
Currently, I think it would be fine this way. You can continue and we see after the changes.

**Comment by kamishettysreeja25** [comment]
@niccokunzmann I have made the required changes as you mentioned earlier.If anything else is required /missing please suggest me .

File: howtoplay.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<html>
+</body>
+</html>
```
Comment (niccokunzmann) [inline howtoplay.html]: you do not need these two tags

File: javascript/main_menu.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	window.location="leaderboard.html";
+			
+	}
+	
+	else if(instance==5)
+	{
+	window.location="links.html";
+			
+	}
+
```
Comment (niccokunzmann) [inline javascript/main_menu.js]: you can remove this file with `git rm `

**Comment by niccokunzmann** [comment]
I commented on some files. You can check and ask back!

**Comment by kamishettysreeja25** [comment]
@niccokunzmann  sorry i could not notice them. i think now it is fine, till if you find any mistake please inform me .I have added the links of the main menu in links.js itself ,removed main_menu.js and corrected the syntax error

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 please walk through the checkpoints at https://github.com/fossasia/flappy-svg/pull/100#issuecomment-185599739
and have a look if all the links work, if no files are in the pull-request that are not needed, remove JavaScript from the links completely.
@kamishettysreeja25 please walk through the checkpoints at https://github.com/fossasia/flappy-svg/pull/100#issuecomment-185599739
and have a look if all the links work, if no files are in the pull-request that are not needed, remove JavaScript from the links completely.

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 I will have a look at it when you notify me :)

**Comment by kamishettysreeja25** [comment]
@niccokunzmann   i am facing a problem in merging ,i am solving it  i will notify you when i am done

**Comment by kamishettysreeja25** [comment]
-[x] consistent indentation in heads
-[x] Create valid HTML code - for ever open tag there should be a closing tag and vice versa.
-[x] favicon used by html files
-[x] titles in html files
-[x]  removed main_menu.js- used direct links wherever necessary 
-[x] "Exit" should to  be "Links"

@niccokunzmann 
Sorry I was late i was facing some problems with merging now it works
This commit has  completed the tasks as you mentioned in the #100 (comment) 
I have added 4 files 
1-main_menu.html - contains the main menu code
2-leaderboard.html-this page is empty - should be developed soon 
3-howtoplay.hmtl- this contains the game rules which should be added
4-main_menu.css - contains the styling for main_menu 	

If anything else is missing please tell me :) 
thank you

**Comment by niccokunzmann** [comment]
@kamishettysreeja25 It looks like you merged the master branch into the gh-pages branch. My fault I made this possible. It contains many more things than which are useful to just play the game. Because the app downloads the game, we moved the source code of the app and some other things to the master branch so the download size of the game is not that big.

Can you start a new pull-request based on the current state in the gh-pages branch + your work?
  
**Example**
Here I outline a plan to create a new pull-request.

create a new branch to work on the main menu

      git checkout -b mainmenu-patch gh-pages

pull the current version of flappy from fossasia

    git pull https://github.com/fossasia/flappy-svg.git gh-pages

pick the commits that you want to have in your new branch e.g. ca46732  (search for ca46732 to find on this page)

    git cherry-pick ca46732
    ..............many more ......

push the changes

    git push origin mainmenu-patch

now you should have this branch on your github repository and be able to create a new pull-request.

I am not a 100% sure this works. It would be what I would try.

**Comment by kamishettysreeja25** [comment]
@niccokunzmann  I have tried it.It is showing all the commits from the master branch 
screenshot from 2016-02-24 12 58 51

**Comment by niccokunzmann** [comment]
Closing in favor of #104.

**Pull request closed without merging**